client/src/scss/themes.scss

```diff
@@ -0,0 +1,60 @@
+@import "./variables";
+
+:root {
+  --body-background: #{$white};
+  --body-color: #{$gray-9};
+  --active-item-color: #{$gray-midnight};
+  --navbar-bg: rgba(255, 255, 255, 0.2);
+  --navbar-color: #{$gray-6};
+  --header-bg: linear-gradient(360deg, #e1e9f9 0.06%, #f3f7ff 99.2%);
+  --bubble-1: url("../assets/bubble-1.svg");
+  --bubble-2: url("../assets/bubble-2.svg");
+  --header-color: #{$gray-6};
+  --header-box-bg: rgba(255, 255, 255, 0.6);
+  --header-label-color: #{$gray-midnight};
+  --header-icon-color: #{$gray-7};
+  --header-svg-bg: #{$gray-2};
+  --border-color: #{$gray-3};
+  --expanded-color: #{$gray-7};
+  --light-bg: #{$gray-1};
+  --input-border-color: #d9e1ef;
+  --input-focus-border-color: #{$gray-7};
+  --modal-color: #{$gray-7};
+  --message-confirmed-bg: #{$mint-green-1};
+  --message-conflicting-bg: rgba(202, 73, 61, 0.24);
+  --qr-color: #{$gray-9};
+  --box-shadow-down: 0 4px 30px rgba(132, 147, 173, 0.1);
+  --header-identity: #{$gray-daylight};
+  --did-color: #{$gray-7};
+  --search-bg-opacity: 0.6;
+  --card-color: #{$gray-6};
+
+  body.darkmode {
+    --body-background: #091326;
+    --body-color: #{$white};
```

Comment:
Change --body-color to #fcfcf - Off White. All of the white text in dark mode is off-white (#fcfcfc) by design.

Reply:
Resolved